### DISCUSSION TITLE
Feedback and Feature Request for Win11 Compatibility

### DISCUSSION THREAD

**Discussion by realczhe** [krlvm/BeautySearch#50] [Ideas]
Hey there,
I recently came across your program, and I'm pretty impressed.
I noticed that there's a solution (https://answers.microsoft.com/en-us/windows/forum/all/how-do-i-remove-suggested-and-top-apps-from-the/2e44dbd2-a3e1-409a-8bc4-33334aca2a36) suggesting the use of it to get rid of those annoying "suggested" and "top apps" from the search bar in Win10. 
I've been trying it with Win11, and it seems like it's not a perfect fit yet.
Is there any chance you could update your program to work with Win11 23H2? It would also be awesome to have an easy way to hide or remove those "suggested" and "top apps" in Win11.(≧ω≦)

**Comment by krlvm** [comment]
I personally don't use Windows 11, so yes, Windows 11 version is definitely out of date.

Yes, those sections can be removed indeed, I will update my Windows 11 VM and implement this soon
